src/Types.purs

```diff
@@ -64,7 +67,6 @@ type Physical = {
   , vel  :: Vector
   , acc  :: Vector
   , size :: Vector
-  , inertia :: Vector
```

Comment:
Inertia will be needed. Trust me.

Reply:
I am not sure what you mean by "inertia" here then.
Normally, inertia is force F = m * a
It is derivative of E = (1/2) * m * (v^2) which is energy of moving object
We can use the energy equation to calculate the resulting velocities of two colliding objects since E_before_collision=E_after_collision

Reply:
What I mean... I would make sense to have _mass_ as a property of Physical.
Unless you have something exotic (i.e. non-Newtonian physics) in mind - simple & fun games are all about that